org-timeblock.el

```diff
@@ -683,7 +683,7 @@ commands"
   (quit-window t))
 
 (defun ot--schedule-time (marker eventp)
-  "Change the timestamp of the org entry at MARKER to a specified time(range) interactively.
+  "Interactively change Org entry timestamp at MARKER to a specified time-range.
 If EVENTP is non-nil, change timestamp of the event.
 
 Schedule or event date won't be changed.
```

Comment:
I think, we need to clarify here that the time might be a timerange which depends on user interactive choice.

Reply:
Left for you to reword as you see fit.